BoxContentSDK/BoxContentSDK/Categories/UIApplication+ExtensionSafeAdditions.m

```diff
@@ -12,18 +12,23 @@ + (UIApplication *)box_sharedApplication
 {
 #ifndef BOX_TARGET_IS_EXTENSION
```

Comment:
yeah but during execution, what would happen when the extension calls those methods ? it would result in calling the forbidden APIs